tools/api-generator/fixtures/02-multiple-content-types.yaml

```diff
@@ -27,123 +27,8 @@ servers:
-    name: Resource Policies
-  - description: Creates one index to a database deployment in a rolling manner. You can't create a rolling index on an `M0` free cluster or `M2/M5` shared cluster.
-    name: Rolling Index
-  - description: Returns details that describe the MongoDB Cloud build and the access token that requests this resource. This starts the MongoDB Cloud API.
-    name: Root
-  - description: Returns, adds, edits, and removes serverless instances.
-    name: Serverless Instances
-  - description: Returns, adds, edits, and removes private endpoints for serverless instances. To learn more, see the Atlas Administration API tab on the following tutorial.
-    externalDocs:
-      description: Set Up a Private Endpoint for a Serverless Instance Tutorial
-      url: https://dochub.mongodb.org/core/serverless-private-endpoint
-    name: Serverless Private Endpoints
-  - description: Endpoints for managing Service Accounts and secrets. Service Accounts are used for programmatic access to the Atlas Admin API through the OAuth 2.0 Client Credentials flow. This feature is available as a Preview feature.
-    externalDocs:
-      description: Get Started with the Atlas Administration API
-      url: https://www.mongodb.com/docs/atlas/configure-api-access/
-    name: Service Accounts
-  - description: Returns and adds restore jobs for shared-tier database deployments.
-    name: Shared-Tier Restore Jobs
-  - description: Returns and requests to download shared-tier database deployment snapshots.
-    name: Shared-Tier Snapshots
-  - description: Returns, adds, edits, and removes Streams Instances. This resource requires your project ID.
-    name: Streams
-  - description: Returns, adds, edits, or removes teams.
-    name: Teams
-  - description: Atlas test endpoints.
-    name: Test
-  - description: |-
-      Returns, adds, edits, and removes third-party service integration configurations. MongoDB Cloud sends alerts to each third-party service that you configure.
-
-      **IMPORTANT**: Each project can only have one configuration per integrationType.
-    name: Third-Party Integrations
-  - description: Returns, edits, and removes user-managed X.509 configurations. Also returns and generates MongoDB Cloud-managed X.509 certificates for database users. The following resources help manage database users who authenticate using X.509 certificates. You can manage these X.509 certificates or let MongoDB Cloud do it for you. If MongoDB Cloud manages your certificates, it also manages your Certificate Authority and can generate certificates for your database users. No additional X.509 configuration is required. If you manage your certificates, you must provide a Certificate Authority and generate certificates for your database users.
-    externalDocs:
-      description: Self-Managed X.509 Certificates
-      url: https://www.mongodb.com/docs/atlas/security-self-managed-x509/
-    name: X.509 Authentication
+    name: Clusters
```

Comment:
Based on the changes in `.snapshots` it looks like you forgot to remove this.

Reply:
I actually added back, this check was actually verifying two things, missing tag and also multiple content types. Since we had already a test for missing tag I've decided to fix this test